scripts/_variables

```diff
@@ -11,4 +11,4 @@ nodejs_version=4
 
 # Version of mypads
 # This variable is mostly used to force an upgrade of the package in case of new versions of mypads.
-mypads_version=1.4.5
+mypads_version=1.4.6
```

Comment:
This variable is not used in the upgrade script, so I suppose the retrieval of etherpad dependencies automatically downloads the latest version of mypads...?

Reply:
All the additional modules are updated here
https://github.com/YunoHost-Apps/etherpad_mypads_ynh/blob/master/scripts/upgrade#L154-L161

As said in the comment before the variable, the idea is mostly to force an upgrade for existing installation.

Reply:
... and it is used to download this specific version in install script.
So you confirm this variable is not used for upgrade, just for install?

Reply:
Yes, only for the install script, but because modifying the version number is going to trig an upgrade, mypads will be upgrade by npm update at the line 160 in the upgrade script.

Reply:
But in my understanding, it's the version number in the manifest (and not the mypads_version) that triggers the upgrade :wink:

Reply:
Yeah sure, I agree that it's quite useless, it's just a way to keep trace of this upgrade.
If I just change the version number in the manifest, who's going to know why I did that ?